h2o-algos/src/main/java/hex/anovaglm/ANOVAGLMModel.java

```diff
@@ -84,7 +84,7 @@ public static class ANOVAGLMParameters extends Model.Parameters {
     public int _nfolds = 0; // disable cross-validation
     public Key<Frame> _plug_values = null;
     public boolean _save_transformed_framekeys = false; // for debugging, save the transformed predictors/interaction
-    public int _nparallelism = 4;
+    public int _nparallelism = H2O.NUMCPUS;
```

Comment:
Why not `H2O.ARGS.nthreads`?

I'm not suggesting that the `nthreads` would be better but I know you can specify `nthreads` in `h2o.init(...)` so I am curious when we should use the `NUMCPUS` and when the `nthreads`. `NUMCPUS` can be set by setting `sys.ai.h2o.activeProcessorCount` property which seems harder than setting `nthreads`.

If we don't use `nthreads` for parallelism, does it make sense to have it exposed in the `h2o.init`?